app/api/v1/endpoints/admin.py

```
@@ -0,0 +1,236 @@
+
+        log_user_action(
+            admin.email,
+            f"created user {user.email} with role {user.role}"
+        )
+
+        user_dict = _build_user_response(user, user_service)
+        user_dict['is_oidc_user'] = False
+
+        return UserResponse.model_validate(user_dict)
+    except UserAlreadyExistsError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)